ui/src/main/webapp/src/app/components/reports/application-details/application-details.component.html

```diff
@@ -58,16 +58,19 @@
 
 <div class="row container-fluid" *ngIf="allProjects?.length > 0">
     <div class="theme-showcase" role="main">
-        <div class="panel panel-primary" *ngFor="let traversal of rootProjects">
-            <div class="panel-heading" *ngIf="(traversal.applicationMessages | async).length > 0">
-                <h3 class="panel-title">Application Messages for: {{traversal.path}}</h3>
+        <template ngFor let-traversal [ngForOf]="rootProjects">
+            <div class="panel panel-primary" *ngIf="(traversal.applicationMessages | async).length > 0">
+                <div class="panel-heading">
+                    <h3 class="panel-title">Application Messages for: {{traversal.path}}</h3>
+                </div>
+                <table class="table table-striped table-bordered" *ngIf="(traversal.applicationMessages | async).length > 0">
+                    <tr *ngFor="let applicationMessage of (traversal.applicationMessages | async)">
```

Comment:
That is correct. This code was a leftover from when it was previously not handled in the parent (which was a mistake).